composer.json

```diff
@@ -22,9 +22,14 @@
     "require": {
         "php": ">=5.3.2",
         "symfony/doctrine-bridge": "~2.1",
+        "symfony/console": "~2.1",
         "doctrine/doctrine-bundle": "~1.0",
+        "doctrine/orm": "~2.0",
```

Comment:
This bundle was working before your modification and your modification is not adding a specific dependency on ORMPurger. Thus, I would advise not to modify this. If this is to be changed, it would need to be in a separate PR.